components/composition/Hidden/Hidden.css

```diff
@@ -0,0 +1,50 @@
+/*
+  Prettier tried to put everything on new lines, but we don't want that in this file.
+  Can't see how to disable just that rule.
+*/
+/* stylelint-disable prettier/prettier */
+
+.diamond-hidden-small-mobile { display: none; }
+.diamond-visible-small-mobile { display: block; }
```

Comment:
[This is an interesting approach](https://medium.com/codyhouse/a-new-approach-to-visibility-utility-classes-based-on-css-variables-eb1e2a01fa3b#:~:text=The%20primary%20goal%20of%20the,a%20breakpoint%20using%20media%20queries.) to control the display through `display: var(--display, block);` but I don't think it works that well here unless we opt into it within every component

Reply:
That's clever because you get the fallback display back. But then it does only work on components. I guess you could set up base elements too.

Reply:
I think because you can use the class to wrap things anyway I don't know that you _need_ to support all the display styles